src/shared/components/HomeToggle/HomeToggle.tsx

```diff
@@ -0,0 +1,46 @@
+'use client';
+import React, { useState } from 'react';
```

Comment:
```suggestion
'use client';

import { useState } from 'react';
```
**Request**: React 삭제해도 돼요!!